docs/source/conf.py

```diff
@@ -91,16 +91,9 @@
 # documentation.
 #
 # html_theme_options = {}
-html_theme_options = {
-    'github_user': 'neuropoly',
-    'github_repo': 'axondeepseg',
-    'github_button': True,
-    'github_banner': True,
-}
-
 html_theme_options = {
     'collapse_navigation': False,
-    'display_version': False,
+    'version_selector': False,
```

Comment:
It looks like my change here introduced a new warning:

```
preparing documents... WARNING: unsupported theme option 'version_selector' given
```

But this is strange to me, because it works locally, and `version_selector` is explicitly mentioned here: https://sphinx-rtd-theme.readthedocs.io/en/stable/configuring.html#confval-version_selector

Reply:
Okay, I think the reason here is that `version_selector` is relatively new:

> `display_version`: Deprecated since version 3.0.0: Removed in favor of `version_selector` and `language_selector`.

Version 3.0.0 was released on PyPI on October 7, 2024. However, we don't use pip/PyPI for RTD, we use conda's `defaults` channel, which is stuck on 2.0.0:

![image](https://github.com/user-attachments/assets/7416d72e-a01c-4829-959f-3583c6cab65c)

So, to stay up to date, we would either need to switch to `conda-forge`, switch our miniconda to miniforge (see also: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4670), or to use `pip` for our RTD builds.

I can just add `conda-forge` for now to get the tests passing. :)

Reply:
> I can just add conda-forge for now to get the tests passing. :)

LGTM!

Reply:
Woof, yeah... agreed!